### PR TITLE
This reverts commit 6eb9f27a3d95bbfe9184c94ecf39741fb42cf1f7

### PR DESCRIPTION
**Pull request by KKoukiou**
When creating dummy ISO files don't use qemu-img, this does not
reflect the reality anyway.

### CODE REVIEW

File: test/check-machines-disks [hunk 1]
```diff
@@ -800,8 +797,8 @@ class TestMachinesDisks(VirtualMachinesCase):
         self.createVm("subVmTest1")
 
         # Prepare file for Custom File disk type
-        m.execute("qemu-img create -f raw /var/lib/libvirt/novell.iso 1M")
-        m.execute("qemu-img create -f qcow2 /var/lib/libvirt/novell.qcow2 1M")
```
Comment (martinpitt) [inline test/check-machines-disks:804]: Was that by accident, or on purpose? I do see that using qemu-img for a fake iso file is wrong, but for a .qcow2 file it still seems right?

File: test/check-machines-settings [hunk 2]
```diff
@@ -253,8 +253,7 @@ class TestMachinesSettings(VirtualMachinesCase):
         b = self.browser
 
         self.createVm("subVmTest1")
-        self.machine.execute("qemu-img create -f raw /var/lib/libvirt/images/phonycdrom 1M; "
-                             "virsh attach-disk subVmTest1 /var/lib/libvirt/images/phonycdrom sdb --type cdrom --config")
+        self.machine.execute("touch /var/lib/libvirt/images/phonycdrom; virsh attach-disk subVmTest1 /var/lib/libvirt/images/phonycdrom sdb --type cdrom --config")
```
Comment (martinpitt) [inline test/check-machines-settings:256]: The two-line version was easier to read/review.

**Review by martinpitt — COMMENTED** [review]
Nice! One question/potential gotcha, otherwise, great debugging work!

File: test/check-machines-disks (hunk 1 above)
Comment (KKoukiou) [reply test/check-machines-disks:804]: I just reverted the commit completely, with git revert. Yes. this should be fine.

File: test/check-machines-settings (hunk 2 above)
Comment (KKoukiou) [reply test/check-machines-settings:256]: I just reverted the commit, but I can also partially revert it. and keep this and the comment from above.

**Review by martinpitt — APPROVED** [review]
I see, thanks!

**Pull request merged**